### ISSUE TITLE
Documentation not building on readthedocs

### ISSUE THREAD

**Issue by anoadragon453** [HelloZeroNet/Documentation#95]
The documentation is failing to build due to a lack of updated mkdocs and mkdocs-material pypi packages, seen here: https://readthedocs.org/projects/zeronet/builds/7896970/

The rtd config is unfortunately not in this repo. Could you update it on the rtd side so it has those changes @hellozeronet?

**Comment by HelloZeroNet** [comment]
The error:
```
WARNING -  Config value: 'nav'. Warning: Unrecognised configuration name: nav 
ERROR   -  Config value: 'theme'. Error: Unrecognised theme name: 'material'. The available installed themes are: readthedocs, mkdocs 

Aborted with 1 Configuration Errors!
```

**Comment by HelloZeroNet** [comment]
Switched back to mkdocs theme, it's building now. Found some problems:
 - The API reference does not lists the commands on the left list: https://zeronet.readthedocs.io/en/latest/site_development/zeroframe_api_reference/
 - The order of the left menu items is changing randomly: https://zeronet.readthedocs.io/en/latest/ (help section on the bottom), https://zeronet.readthedocs.io/en/latest/site_development/zeroframe_api_reference/ (help section on the top)

**Comment by anoadragon453** [comment]
> The API reference does not lists the commands on the left list

Ah, this is a problem with the titles. Let me make a PR.

> The order of the left menu items is changing randomly

I think this is fixed with the latest version of mkdocs.

**Comment by HelloZeroNet** [comment]
> I think this is fixed with the latest version of mkdocs.

We have no access the version of the mkdocs on readthedocs.org, so probably we have to revert back changes in mkdocs.yml

**Comment by anoadragon453** [comment]
Ahh, that's too bad. Alright. We can do that.

Any chance of docs.zeronet.io at some point and just host the static docs files ourselves?

**Comment by anoadragon453** [comment]
Essentially you'll want to set up a webhook for whenever master is pushed to. This will then send out a notification whenever that happens, which a program like https://github.com/OllieJones/git-webhook-responder/blob/master/README.md can respond to and run a command, like `mkdocs build`, on.

**Comment by anoadragon453** [comment]
Closed in favor of #101
